package.json

```diff
@@ -1,18 +1,19 @@
 {
   "packageManager": "pnpm@7.27.1",
-  "name": "turbo",
+  "name": "webstudio",
   "version": "0.0.0",
   "private": true,
   "scripts": {
     "": "For some reason remix build does not finish when run with turbo or pnpm recursively",
-    "build-packages": "pnpm run --recursive --parallel --filter='!./apps/*' --filter='!./packages/sdk-size-test' build",
-    "build": "pnpm build-packages && pnpm --filter=@webstudio-is/sdk-size-test build && pnpm --filter=@webstudio-is/builder build",
+    "build": "turbo run build",
```

Comment:
Turborepo calculates the ordering for you. No filter required because of the renamed script.